impressive_strawberry/web/models/full.py

```diff
@@ -97,7 +97,7 @@ class UserFull(read.UserRead):
     """
 
     application: read.ApplicationRead
-    unlocks: t.List[read.AchievementRead]
+    unlocks: t.List[UnlockFull]
```

Comment:
Per funzionare dovrebbe funzionare, ma potrebbe essere pericoloso se iniziamo a referenziare anche altre cose in `UnlockFull`.

Chissà se c'è un modo più ordinato di farlo...